### PR TITLE
Updated cm6-graphql package README

### PR DESCRIPTION
**Pull request by imolorhe**
Updated cm6-graphql package README

### CODE REVIEW

File: packages/cm6-graphql/README.md [hunk 1]
```diff
@@ -1 +1,47 @@
...
+  extensions: [
+    basicSetup,
+    graphql(myGraphQLSchema),
+  ],
+  parent: document.body
+})
+
+```
+
+Note: You have to provide a theme to codemirror 6 for the styling you want. You can take a look at [this example](https://github.com/graphql/graphiql/blob/main/examples/cm6-graphql-parcel/src/index.ts) or see the codemirror 6 [documentation examples](https://codemirror.net/examples/styling/) for more details.
```
Comment (TallTed) [inline packages/cm6-graphql/README.md:35]:
```suggestion
Note: You have to provide a theme to CodeMirror 6 for the styling you want. You can take a look at [this example](https://github.com/graphql/graphiql/blob/main/examples/cm6-graphql-parcel/src/index.ts) or see the CodeMirror 6 [documentation examples](https://codemirror.net/examples/styling/) for more details.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Fix some capitalization

**Comment by acao** [comment]
thank you @imolorhe ! just need to run `yarn pretty` to make prettier happier about the markdown

one other thing while we're here, can we add a section at the beginning of the `codemirror-graphql` readme that says something like "for codemirror 6 support, use cm6-graphql" with a link?

File: packages/cm6-graphql/README.md [hunk 2]
```diff
@@ -5,43 +5,48 @@
...
+});
 ```
 
-Note: You have to provide a theme to CodeMirror 6 for the styling you want. You can take a look at [this example](https://github.com/graphql/graphiql/blob/main/examples/cm6-graphql-parcel/src/index.ts) or see the CodeMirror 6 [documentation examples](https://codemirror.net/examples/styling/) for more details.
+Note: You have to provide a theme to CodeMirror 6 for the styling you want. You
+can take a look at
+[this example](https://github.com/graphql/graphiql/blob/main/examples/cm6-graphql-parcel/src/index.ts)
+or see the CodeMirror 6
+[documentation examples](https://codemirror.net/examples/styling/) for more
+details.
```
Comment (TallTed) [inline packages/cm6-graphql/README.md:39]:
```suggestion
_**Note:** You have to provide a theme to CodeMirror 6 for the styling you want. You
can take a look at
[this example](https://github.com/graphql/graphiql/blob/main/examples/cm6-graphql-parcel/src/index.ts)
or see the CodeMirror 6
[documentation examples](https://codemirror.net/examples/styling/) for more
details._
```

File: packages/cm6-graphql/README.md [hunk 3]
```diff
@@ -5,43 +5,48 @@
...
 
 ### Updating schema
 
-If you need to update the GraphQL schema used in the editor dynamically, you can call `updateSchema` with the CodeMirror `EditorView` instance and the new schema
+If you need to update the GraphQL schema used in the editor dynamically, you can
```
Comment (TallTed) [inline packages/cm6-graphql/README.md:43]:
```suggestion
If you need to dynamically update the GraphQL schema used in the editor, you can
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Review by acao — APPROVED** [review]
awesome work!

**Comment by TallTed** [comment]
@acao — You merged without you or @imolorhe applying or even commenting on [my suggested changes](https://github.com/graphql/graphiql/pull/3127#pullrequestreview-1375167866). Why?

**Comment by TallTed** [comment]
I've taken my other suggestions to #3174

**Pull request merged**